src/MonitoringFunctions.Common/TeamsConnector.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace MonitoringFunctions.Common
+{
+    /// <summary>
+    /// Represents a connector for MS Teams.
+    /// </summary>
+    internal sealed class TeamsConnector
+    {
+        private static readonly HttpClient HttpClient = new HttpClient();
```

Comment:
`HttpClient` is disposable.

https://docs.microsoft.com/en-us/dotnet/api/system.net.http.httpclient.dispose?view=net-5.0

Making matters more interesting, [C#: HttpClient should NOT be disposed](https://medium.com/@nuno.caneco/c-httpclient-should-not-be-disposed-or-should-it-45d2a8f568bc)

Reply:
I decided to keep it undisposed as discussed in the article.